### PR TITLE
[14.0][MIG]product_cost_security

### PR DESCRIPTION
**Pull request by watthanun**

### CODE REVIEW

**Review by tonygalmiche — APPROVED** [review]
LGTM : Functional review only with runbot

**Review by CasVissers-360ERP — APPROVED** [review]
LGTM, tested in migrated DB.

Maybe this comment could be removed:

File: product_cost_security/README.rst [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fproduct--attribute-lightgray.png?logo=github
+    :target: https://github.com/OCA/product-attribute/tree/13.0/product_cost_security
+    :alt: OCA/product-attribute
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/product-attribute-13-0/product-attribute-13-0-product_cost_security
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/135/13.0
```
Comment (sergio-teruel) [inline product_cost_security/README.rst]: Yo must re-generate this file...

File: product_cost_security/views/product_views.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="product_template_form_view" model="ir.ui.view">
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_form_view" />
+        <field name="arch" type="xml">
+            <!-- To hide uom_name field, new in v13.0 -->
```
Comment (sergio-teruel) [inline product_cost_security/views/product_views.xml]: Remove this comment

**Review by sergio-teruel — CHANGES_REQUESTED** [review]

**Review by sergio-teruel — APPROVED** [review]

**Pull request merged**